backend/dissemination/models.py

```diff
@@ -384,189 +228,124 @@ class General(models.Model):
     )
     auditor_email = models.TextField(
         "CPA mail address (optional)",
-        null=True,
         help_text=docs.auditor_email,
     )
     auditor_firm_name = models.TextField(
         "CPA Firm Name", help_text=docs.auditor_firm_name
     )
     # Once loaded, would like to add these as regular addresses and just change this to a country field
-    auditor_foreign_addr = models.TextField(
+    auditor_foreign_address = models.TextField(
         "CPA Address - if international",
-        null=True,
         help_text=docs.auditor_foreign,
     )
     auditor_ein = models.TextField(
         "CPA Firm EIN (only available for audit years 2013 and beyond)",
-        null=True,
         help_text=docs.auditor_ein,
     )
 
     # Agency
     cognizant_agency = models.TextField(
         "Two digit Federal agency prefix of the cognizant agency",
-        null=True,
         help_text=docs.cognizant_agency,
     )
     oversight_agency = models.TextField(
         "Two digit Federal agency prefix of the oversight agency",
-        null=True,
         help_text=docs.oversight_agency,
     )
 
     # Dates
-    initial_date_received = models.DateField(
+    date_created = models.DateField(
         "The first date an audit component or Form SF-SAC was received by the Federal audit Clearinghouse (FAC).",
-        null=True,
         help_text=docs.initial_date_received,
     )
     ready_for_certification_date = models.DateField(
         "The date at which the audit transitioned to 'ready for certification'",
-        null=True,
     )
     auditor_certified_date = models.DateField(
-        "The date at which the audit transitioned to 'auditor certified'", null=True
+        "The date at which the audit transitioned to 'auditor certified'",
     )
     auditee_certified_date = models.DateField(
-        "The date at which the audit transitioned to 'auditee certified'", null=True
+        "The date at which the audit transitioned to 'auditee certified'",
     )
     certified_date = models.DateField(
-        "The date at which the audit transitioned to 'certified'", null=True
+        "The date at which the audit transitioned to 'certified'",
     )
     submitted_date = models.DateField(
-        "The date at which the audit transitioned to 'submitted'", null=True
-    )
-    auditor_signature_date = models.DateField(
-        "The date on which the auditor signed the audit", null=True
-    )
-    auditee_signature_date = models.DateField(
-        "The date on which the auditee signed the audit", null=True
-    )
-    fy_end_date = models.DateField(
-        "Fiscal Year End Date", null=True, help_text=docs.fy_end_date
+        "The date at which the audit transitioned to 'submitted'",
     )
+    # auditor_signature_date = models.DateField(
+    #     "The date on which the auditor signed the audit",
+    # )
+    # auditee_signature_date = models.DateField(
+    #     "The date on which the auditee signed the audit",
+    # )
```

Comment:
@jadudm after clearing up the dates issue, do these signature fields need to be added back in?

Reply:
I... think we only need to disseminate the field that is auto-generated. 

We can talk about that, but I'm not sure if we need to disseminate the name, role, and date from that form. (We capture it elsewhere, right?) However, given that the user can enter anything they want in that field, and what we care about is the system timestamp, I think the date we disseminate is the one that is auto-recorded.